react-components/stories/utilities/RevealStoryContainer.tsx

```diff
@@ -0,0 +1,69 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { useRef, type ReactElement, type ReactNode, useMemo } from 'react';
+import { RevealKeepAliveContext } from '../../src/components/RevealKeepAlive/RevealKeepAliveContext';
+import { RevealContainer } from '../../src/components/RevealContainer/RevealContainer';
+import { type FdmNodeCache } from '../../src/components/NodeCacheProvider/FdmNodeCache';
+import { type AssetMappingCache } from '../../src/components/NodeCacheProvider/AssetMappingCache';
+import { type CogniteClient } from '@cognite/sdk';
+import { type Color } from 'three';
+import { Cognite3DViewer, type Cognite3DViewerOptions } from '@cognite/reveal';
+import { createSdkByUrlToken } from './createSdkByUrlToken';
+
+type RevealStoryContainerProps = {
+  sdk?: CogniteClient;
+  viewer?: Cognite3DViewer;
+  color?: Color;
+  appLanguage?: string;
+  children?: ReactNode;
+  viewerOptions?: Pick<
+    Cognite3DViewerOptions,
+    | 'antiAliasingHint'
+    | 'loadingIndicatorStyle'
+    | 'rendererResolutionThreshold'
+    | 'antiAliasingHint'
+    | 'ssaoQualityHint'
+    | 'pointCloudEffects'
+    | 'enableEdges'
+  >;
+};
```

Comment:
Can we use the props-type for `RevealContainer` to make this easier to maintain?